.github/workflows/ci.yml

```diff
@@ -75,18 +75,18 @@ jobs:
         java-version: 11
     - name: Run integration tests
       run: |
-        cd test-project
+        cd src\test-project
         chmod +x gradlew.bat
         .\gradlew.bat clean test --warning-mode none
       shell: powershell
     - uses: actions/upload-artifact@v2
       with:
         name: ${{ runner.os }}-test-reports
-        path: test-project/build/reports/tests/test
+        path: src\test-project/build/reports/tests/test
```

Comment:
any particular reason to use a backslash while all other separators are forward slashes?

Reply:
Ups, I have overlooked that. Fixed.